ras_party/controllers/queries.py

```diff
@@ -67,6 +67,26 @@ def query_business_respondent_by_respondent_id_and_business_id(business_id, resp
     return response
 
 
+def query_change_respondent_details(respondent_data, session):
```

Comment:
I think the method name update_respondent_details is good enough?

Reply:
I prefer this to having the work query in the name as well, the word query is in the name of several other functions in this file however